### ISSUE TITLE
passing in `all_folder_admins` var without `per_folder_admins` does nothing

### ISSUE THREAD

**Issue by max-l-weaver** [terraform-google-modules/terraform-google-folders#69]
### TL;DR

You're supposed to be able to pass in a list of `all_folder_admins` which should create the folder bindings accordingly. This is not happening due to the conditional inside the folder binding resource.

### Expected behavior

You pass in a list of folder admins, i.e.
```
all_folder_admins = [
    "group:admins@bob.com"
]
```
and you see on plan/apply folder binding resources with the above values.

### Observed behavior

passing the aforementioned variable was as ineffective as Ranier Wolfcastle's goggles during the tsunami of acid scene in Radioactive Man: The Movie.

Looking at the Terraform module code it appears that the local value `local.folder_admin_roles_map_data` expects `var.per_folder_admins` to be set. This local is a requirement for the resource `google_folder_iam_binding.owners`, however if you just pass in the list of roles for `all_folder_admins` and nothing for the aforementioned variable the folder binding is completely ignored.

### Terraform Configuration

```hcl
module "folder" {
  source             = "terraform-google-modules/folders/google"
  version            = "4.0.0"
  parent             = "folder/12345667"
  set_roles          = true
  all_folder_admins  = ["group:admins@bob.com"]
  names              = ["bob_folder"]
}
```

### Terraform Version

```sh
`v1.5.2`
```

### Additional information

The docs for `all_folder_admins` specifies: List of roles that will be applied to a folder if roles are not explictly specified in per_folder_admins.

This suggests that if per_folder_admins is not passed in this should be properly handled, but as you can see in the Terraform if `per_folder_admins` is not passed in the iam binding resource `for_each` just sees an empty map.

**Comment by max-l-weaver** [comment]
Is anyone actually maintaining this module? If not I'll submit a PR myseld
